cmd/flow-rosetta-server/README.md

```diff
@@ -47,5 +48,5 @@ The following command line starts the Flow DPS Rosetta server for a main network
 It uses a local instance of the Flow DPS Server for access to the execution state.
 
 ```sh
-./flow-dps-rosetta -a "127.0.0.1:5005" -p 8080
+./flow-dps-rosetta -a "127.0.0.1:5005"
```

Comment:
The address here should be with port `8008`, and we should add the Flow DPS dependency at least.